src/pybind/mgr/dashboard/services/rbd.py

```diff
@@ -101,6 +101,14 @@ def _list(ioctx):
                 with rbd.Image(ioctx, self._image_name) as image:
                     result = image.config_list()
             else:  # pool config
+                pg_status = CephService.get_pool_pg_status(self._pool_name)
+                if 'creating+incomplete' in pg_status:
+                    # If config_list would be called with ioctx if it's a bad pool,
+                    # the dashboard would stop working, waiting for the response
+                    # that would not happen.
+                    #
+                    # This is only a workaround for https://tracker.ceph.com/issues/43771
+                    return []
```

Comment:
I also described that changes through this PR should be reverted, if https://tracker.ceph.com/issues/44224 is implemented.

Reply:
So that PR won't go off the radar until this workaround is removed. Is that it? I simply say that TODO comments come but then never leave, so having a specific ticket open is the best (only) way to ensure we are aware of this 'debt'.

Reply:
I will also add the new ticket to that comment.